recipes/poselib/all/conanfile.py

```diff
@@ -76,15 +76,18 @@ def source(self):
 
     def generate(self):
         tc = CMakeToolchain(self)
+        tc.cache_variables["MARCH_NATIVE"] = False
```

Comment:
I think this should be `True` based on @vlarsson comment:
> I've also added a flag to disable -march=native in the cmake which should be done for the wheels.
https://github.com/PoseLib/PoseLib/pull/105#issue-2448406439

Reply:
No, we should have march_native to be false for the wheels.

Reply:
Shouldn't it be kept as disabled to build the Conan binaries for the lowest common denominator architecture by default.
The user is still able to pass `-march=native` as a build flag to any built package through config options in Conan regardless of this.

Reply:
Great, makes sense!